src/mod/server.mod/servmsg.c

```diff
@@ -1058,7 +1083,9 @@ static struct dcc_table SERVER_SOCKET = {
 
 static void server_activity(int idx, char *msg, int len)
 {
-  char *from, *code;
+  char *from, *code, *tag = 0;
```

Comment:
*tag = NULL instead to signify it's the NULL pointer

Reply:
Done